nuxeo-dm/nuxeo-platform-webapp-core/src/main/java/org/nuxeo/ecm/webapp/helpers/StartupHelper.java

```diff
@@ -160,15 +167,18 @@ public String initDomainAndFindStartupPage(String domainTitle, String viewId) {
                 return result;
             }
 
-            String query = "SELECT * FROM Domain WHERE " + NXQL.ECM_MIXINTYPE + " <> '"
-                    + FacetNames.HIDDEN_IN_NAVIGATION + "' AND " + NXQL.ECM_LIFECYCLESTATE + " <> '"
-                    + LifeCycleConstants.DELETED_STATE + "'" + " AND ecm:isCheckedInVersion = 0 "
-                    + " AND ecm:isProxy = 0 ";
-            DocumentModelList domains = documentManager.query(query);
-            if (domains.size() == 1) {
+            Map<String, Serializable> properties = new LinkedHashMap<>();
+            properties.put(CoreQueryAndFetchPageProvider.CORE_SESSION_PROPERTY, (Serializable) documentManager);
```

Comment:
You can use `Collections.singletonMap(CoreQueryAndFetchPageProvider.CORE_SESSION_PROPERTY, (Serializable) documentManager)`.

Reply:
Done